frontend/test/fixture/user/globalSetting.js

```diff
@@ -0,0 +1,6 @@
+import { ROLE } from 'tracim_frontend_lib'
+
+export const globalSetting = {
```

Comment:
rename globalManagerSetting
and rename file too

Reply:
done